bot/lib/bot.py

```diff
@@ -0,0 +1,157 @@
+import discord
+from discord.ext import commands
+import json, requests, sys, argparse, logging
+from . import dummylog, utils
+
+### Get Command Line Arguments and configuration
+description = """Discord Chess Bot is a simple application to play againt other server
+members or to play against the Stockfish AI. Use -h or --help for information about the bot."""
+
+argparser = argparse.ArgumentParser(description=description)
+argparser.add_argument("--file", "--config", dest="configPath", default="./config.json", help = "File Path to Config File")
+argparser.add_argument("-u", "--host", dest="url", default="http://localhost:5000/", help="url for acessing discord bot rest api")
```

Comment:
>abbreviations are actually not necessary for argparse since unambiguous abbreviations are accepted anyways (unless you want abbreviations to show up in the help message i believe). why use "--quiet" and "--no-log" instead of "-q"+"--no-log" or "-q"+"--quiet"?

Good to know I will remove the abbreviations.